### PR TITLE
Issue 19219 graphql fire rules

### PR DESCRIPTION
**Pull request by dsilvam**
A `fireRules` parameter was added to the `page` graphql field. 
Postman tests were added for both `true` and `false` values of the parameter.

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java [hunk 1]
```diff
@@ -35,14 +38,19 @@ public Contentlet get(final DataFetchingEnvironment environment) throws Exceptio
             final HttpServletRequest request = ((DotGraphQLContext) environment.getContext())
                     .getHttpServletRequest();
 
+            final HttpServletResponse response = ((DotGraphQLContext) environment.getContext())
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

File: dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

**Review by nollymar — COMMENTED** [review]
Please add a description field for each postman test that explains what is being tested...it will be helpful for a developer in case any of these tests fail

File: dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

File: dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java [hunk 2]
```diff
@@ -35,16 +38,22 @@ public Contentlet get(final DataFetchingEnvironment environment) throws Exceptio
...
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java:41]: Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)

**Pull request closed without merging**